static/docs/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>LavinMQ HTTP API</title>
+    <script src="https://unpkg.com/@stoplight/elements/web-components.min.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements/styles.min.css">
```

Comment:
It is probably a good idea lock these to a certain version so they does not break on us unexpectedly. Using Subresource Integrity (SRI) we can also ensure that it is the correct version served from the CDN: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

Reply:
or download locally?

Reply:
https://github.com/cloudamqp/lavinmq/pull/693/commits/e91b966906ab4ac3f28329edd91e106fa7579ccc